nipyapi/utils.py

```diff
@@ -57,17 +57,19 @@ def dump(obj, mode='json'):
             obj=prepared_obj,
             sort_keys=True,
             indent=4
-            # default=_json_default
         )
     except TypeError as e:
         raise e
     if mode == 'json':
         return out
     if mode == 'yaml':
```

Comment:
what does `safe` imply?  if it makes guarantees about behavior, maybe add a comment about that to go with the other comments?